js_modules/dagster-ui/packages/ui-core/src/nav/__tests__/useRepositoryLocationReload.test.tsx

```diff
@@ -134,7 +134,8 @@ describe('useRepositoryReloadLocation', () => {
     });
   });
 
-  it('waits for polling when attempting reload', async () => {
+  // eslint-disable-next-line jest/no-disabled-tests
+  it.skip('waits for polling when attempting reload', async () => {
```

Comment:
if I knew why it was flaking I would fix it

Reply:
In what scenario do we expect this to get addressed instead of skipped and ignored indefinitely? 

I feel like deletion is the more honest representation of the choice to remove the coverage this test case provided in response to the flake

Reply:
If we get an issue that this case would have caught then we'd fix it. Figured it would be better to leave it rather than starting from scratch but I'm okay with deleting it too.

Reply:
the actual consequence of either choice is pretty minimal so you can do whatever, I just find the thinking behind the decision interesting

the cost of keeping the dead code around is low - probably just the friction of updating it if something inside it gets refactored and it starts spewing type errors 

if we did hit an issue that this test would have caught, would you want to start from something that is flakey for unknown reasons or start fresh? Whats the costs of having to confront that choice at all?

thanks for coming to my ted talk

Reply:
I mean it's flakey so it does work (sometimes), just there's a timing issue to figure out. So I think there is some value in it.